docs/tutorials/geospatial.ipynb

```diff
@@ -0,0 +1,343 @@
+    "\n",
+    "<center>\n",
+    "<figure>\n",
+    "  <img src=\"https://scitools.org.uk/cartopy/docs/latest/_images/projections-7.png\">\n",
+    "  <figcaption>Mercator</figcaption>\n",
+    "</figure>\n",
+    "</center>\n",
+    "\n",
+    "<center>\n",
+    "<figure>\n",
+    "  <img src=\"https://scitools.org.uk/cartopy/docs/latest/_images/projections-2.png\">\n",
+    "  <figcaption>Albers Equal Area</figcaption>\n",
+    "</figure>\n",
+    "</center>\n",
+    "\n",
+    "<center>\n",
+    "<figure>\n",
+    "  <img src=\"https://scitools.org.uk/cartopy/docs/latest/_images/projections-17.png\">\n",
+    "  <figcaption>Interrupted Goode Homolosine</figcaption>\n",
+    "</figure>\n",
+    "</center>\n",
+    "\n",
+    "There are literally thousands of different projections out there, and every dataset (or even different images within a single dataset) can have different projections. Even if you correctly georeference images during indexing, if you forget to project them to a common CRS, you can end up with rotated images with nodata values around them, and the images will not be pixel-aligned.\n",
+    "\n",
+    "<center>\n",
+    "<img src=\"https://pytorch.org/assets/images/torchgeo-reproject.png\", width=\"800\">\n",
+    "</center>\n",
+    "\n",
+    "We can use a command like:\n",
+    "```\n",
+    "$ gdalwarp -s_srs EPSG:5070 -t_srs EPSG:4326 src.tif dst.tif\r",
+    "```\n",
+    "to reproject a file from one CRS to another.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0362b1e-1c47-4884-a414-699db82acb6e",
+   "metadata": {},
+   "source": [
+    "### Resampling\n",
+    "\n",
+    "As previously mentioned, each dataset may have its own unique spatial resolution, and even separate bands (channels) in a single image may have different resolutions. All data (including input images and target masks for semantic segmentation) must be resampled to the same resolution. This can be done using GDAL like so:\n",
+    "```\n",
+    "$ gdalwarp -tr 30 30 src.tif dst.tif\r",
+    "```\n",
+    "Just because two files have the same resolution does not mean that they have *target-aligned pixels* (TAP). Our goal is that every input pixel is perfectly aligned with every expected output pixel, but differences in geolocation can result in masks that are offset by half a pixel from the input image. We can ensure TAP by adding the `-tap` flag:\n",
+    "```\n",
+    "$ gdalwarp -tr 30 30 -tap src.tif dst.tif\r",
+    "```\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0c499d9-e266-4619-863f-e416cc823c58",
+   "metadata": {},
+   "source": [
+    "### Rasterization\n",
+    "\n",
+    "Not all geospatial data is raster data. Many files come in vector format, including points, lines, and polygons.\n",
+    "\n",
+    "<center>\n",
+    "<img src=\"https://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/Bitmap_VS_SVG.svg/2560px-Bitmap_VS_SVG.svg.png\", width=\"500\">\n",
```

Comment:
This is not displayed properly in the notbeook

Reply:
Turns out that Jupyter Notebook is much more forgiving than nbmake. I think the issue here was the comma, tried removing it. Same below. There were also a lot of issues with code cells, let's see if an additional newline helps.